files/paths_stats.md

```diff
@@ -0,0 +1,52 @@
+## Path Performance Information
```

Comment:
Per our definitions in our glossary for DPA, a path is something a person takes; a route is a vehicle.  In real life though, the difference is ambiguous.  Wondering if we should use a term that makes it obvious that is a collection....like collection, pooled, assembly, medley, group etc.

Reply:
how about group or route_group?

Reply:
sounds good